src/app/ApplicationTemplate.Mobile/Android/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
 		  android:versionName="1.0.0">
 
 	<uses-sdk android:minSdkVersion="26"
```

Comment:
Shouldn't we raise the minimum version to android 10 (29)?

Reply:
Let me confirm with management which version should be applied for the min version

Reply:
I'm going to do the change in another PR